docs/datastore/alloydb.md

```diff
@@ -28,16 +28,24 @@
                            vpcaccess.googleapis.com \
                            aiplatform.googleapis.com
     ```
-1. Download and install [postgres-client cli (`psql`)][install-psql].
 
-[install-psql]: https://www.timescale.com/blog/how-to-install-psql-on-mac-ubuntu-debian-windows/
+1. [Install python][install-python] and set up python [virtual environment][venv].
 
-1. Clone this repo to your local machine:
+1. Make sure you have python version 3.10 and up.
 
     ```bash
-    git clone https://github.com/GoogleCloudPlatform/database-query-extension.git
+    python -V
     ```
 
+1. Download and install [postgres-client cli (`psql`)][install-psql].
+
+1. Download and install [jq][install-jq].
```

Comment:
See comment on bug to use a different gcloud command instead of adding this additional tool